topics/statistics/tutorials/FNN/tutorial.md

```diff
@@ -0,0 +1,457 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to feedforward neural networks (FNN)
+zenodo_link: https://zenodo.org/record/4660497
+questions:
+- What is a feedforward neural network (FNN)?
+- What are some applications of FNN?
+objectives:
+- Understand the inspiration for neural networks
+- Learn various activation functions, and classification/regression problems solved by neural networks
+- Discuss various cost/loss functions and the backpropagation algorithm
+- Learn how to create a neural network using Galaxy's deep learning tools
+- Solve a simple regression problem, car purchase price prediction, via FNN in Galaxy
+requirements:
```

Comment:
Is there a logical ordering for the three tutorials you've added? If so it could be helpful to learners if we suggest to them "ok, you've done this, now do this next"? you can specify follow_up_training in a similar way to these requirements.

Reply:
Yes, in this order. 1) Feedforward, 2) Recurrent, and 3) Convolutional. I will update the requirement section of each tutorial to reflect that.